ledger-tool/src/bigtable.rs

```diff
@@ -584,6 +830,124 @@ impl BigTableSubCommand for App<'_, '_> {
                                 .takes_value(false)
                                 .help("Display the full transactions"),
                         ),
+                )
+                .subcommand(
+                    SubCommand::with_name("copy")
+                        .about("copy blocks from a bigtable to another bigtable")
+                        .arg(
+                            Arg::with_name("source_credential_path")
+                                .long("source-credential-path")
+                                .value_name("SOURCE_CREDENTIAL_PATH")
+                                .takes_value(true)
+                                .help(
+                                    "Source bigtable creds path. (could be readonly)",
+                                ),
+                        )
+                        .arg(
+                            Arg::with_name("source_endpoint")
+                                .long("source-endpoint")
+                                .value_name("SOURCE_ENDPOINT")
+                                .takes_value(true)
+                                .help(
+                                    "Source bigtable endpoint.",
+                                ),
+                        )
```

Comment:
We only need the endpoint if the source is an emulator, correct?
How about we replace the two Args with just one:
```suggestion
                        .arg(
                            Arg::with_name("source_emulator_endpoint")
                                .long("source-emulator")
                                .value_name("SOURCE_EMULATOR_ENDPOINT")
                                .takes_value(true)
                                .help(
                                    "Source bigtable is an emulator, found at provided endpoint.",
                                ),
                        )
```
(Disregard comments on `source_is_emulator` if we go this route)

Reply:
this could arg could then conflict with the credential path arg